File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/mad/Java/Go/s466963472_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

var sc = bufio.NewScanner(os.Stdin)
var d int

func read() bool {
	if !sc.Scan() {
		return false
	}
	text := sc.Text()
	d, _ = strconv.Atoi(strings.TrimSpace(text))
	return true
}

func solve() int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		y := x * x
		ans += y * d
	}
	return ans
}

func main() {
	for read() {
		fmt.Println(solve())
	}
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/comment/comment/direct/Java/C/s466963472_commented.c
#include <stdio.h>

int d;

int read() {
    if (scanf("%d", &d) != 1) return 0;
    return 1;
}

int solve() {
    int ans = 0;
    for (int i = 1; i * d < 600; i++) {
        int x = i * d;
        int y = x * x;
        ans += y * d;
    }
    return ans;
}

int main() {
    while (read()) {
        printf("%d\n", solve());
    }
    return 0;
}
// End of Code
